experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -3650,6 +3650,33 @@ def test_qa_status_badge_class(self, qa_status, expected_badge_class):
         )
         self.assertEqual(experiment.qa_status_badge_class, expected_badge_class)
 
+    @parameterized.expand(
+        [
+            [NimbusExperiment.QATestType.FULL, "Full"],
+            [NimbusExperiment.QATestType.SMOKE, "Smoke"],
+            [NimbusExperiment.QATestType.SELF, "Self"],
+            [NimbusExperiment.QATestType.REGRESSION, "Regression"],
+        ]
+    )
```

Comment:
This should be equivalent and cover any future QA test types being added.
```suggestion
    @parameterized.expand(NimbusExperiment.QATestType.choices)
```